### PR TITLE
Curl-lee shuffle fixes due to Windows.

### PR DESCRIPTION
**Pull request by c3r1c3**
Clean-up on isle MAC.
Clean-up (files) on Windows.
More Linux stuff (including CMAKE, but still woefully complete).

### CODE REVIEW

File: linux/EditorWidget-linux.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	VstRect* vstRect = nullptr;
+	effect->dispatcher(effect, effEditGetRect, 0, 0, &vstRect, 0);
+	if (vstRect)
+	{
+		widget->resize(vstRect->right - vstRect->left, vstRect->bottom - vstRect->top);
+	}
+}
+
+void EditorWidget::handleResizeRequest(int width, int height) {
+	// We don't have to do anything here as far as I can tell. The widget will resize the HWIND itself and then
```
Comment (DDRBoxman) [inline linux/EditorWidget-linux.cpp:51]: Does this apply to linux too?

File: obs-vst.cpp [hunk 2]
```diff
@@ -130,27 +131,27 @@ static void fill_out_plugins(obs_property_t *list)
...
 		// If VST3 support is added....
-		// << "C:/Program Files/Common Files/VST3";
-	#elif LINUX
-		dir_list << "/usr/lib/vst"
-		dir_list << "/usr/lib/lxvst"
-		dir_list << "/usr/local/lib/vst"
-		dir_list << "/usr/local/lib/lxvst"
-		dir_list << "~/.vst"
-		dir_list << "~/.lxvst";
+		// << "C:/Program Files/Common Files/VST3/";
```
Comment (DDRBoxman) [inline obs-vst.cpp:143]: We can go ahead and remove this since VST3 uses a different SDK

**Review by DDRBoxman — CHANGES_REQUESTED** [review]

File: linux/EditorWidget-linux.cpp (hunk 1 above)
Comment (c3r1c3) [reply linux/EditorWidget-linux.cpp:51]: I'm trying to find out. The linux stuff isn't complete by a long shot, but it's 'better' in that it compiles and runs, it just crashes when one actually selects a vst plugin.

File: obs-vst.cpp (hunk 2 above)
Comment (c3r1c3) [reply obs-vst.cpp:143]: I promise to have that pulled in my next commit.

(Locally it's already been pulled).

**Pull request merged**